src/lowlevel.ts

```diff
@@ -51,6 +51,50 @@ export function errorWithCause(code: string, cause: Error): Error {
   return new Error(`${code}: ${cause.message}`, { cause });
 }
 
+const warningSymbol = String.fromCharCode(0x26a0);
+const hints = [
```

Comment:
Below is the bulk of the new feature concept.  Right now it just hints for bad syntax and missing column errors.  If this is well accepted we can continually add to hints that the need arises.